build_tools/travis/install.sh

```diff
@@ -25,12 +25,18 @@ if [[ ! -d /home/travis/miniconda3 ]]
     export PATH=/home/travis/miniconda3/bin:$PATH
     echo $PATH
     conda update --yes conda
-    conda create -n testenv --yes python=3.5
+    # If we are running pylint, use Python 3.5.2 due to
```

Comment:
Are you sure this will work?  Seems like you would need to create a `testenv352`, and activate one or the other based on `$RUN_PYLINT`.  The testenv is cached and only created once.